Sources/LiveKit/Core/Room+SignalClientDelegate.swift

```diff
@@ -135,6 +135,46 @@ extension Room: SignalClientDelegate {
         }
     }
 
+    func signalClient(_: SignalClient, didReceiveRoomMoved response: Livekit_RoomMovedResponse) async {
+        log("didReceiveRoomMoved to room: \(response.hasRoom ? response.room.name : "unknown")")
+
+        // Update room info if available
+        if response.hasRoom {
+            _state.mutate {
+                $0.metadata = response.room.metadata
+                $0.isRecording = response.room.activeRecording
+                $0.numParticipants = Int(response.room.numParticipants)
+                $0.numPublishers = Int(response.room.numPublishers)
+            }
```

Comment:
Other fields like sid / name / maxParticipants / creationTime etc. could be missing ?